core/src/test/java/io/confluent/rest/SaslTest.java

```diff
@@ -46,30 +60,36 @@
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.HttpHeaders;
 import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
 import javax.ws.rs.core.SecurityContext;
 
 import io.confluent.common.metrics.KafkaMetric;
 import io.confluent.rest.annotations.PerformanceMetric;
 
+import static javax.ws.rs.core.Response.Status.FORBIDDEN;
+import static javax.ws.rs.core.Response.Status.OK;
+import static javax.ws.rs.core.Response.Status.UNAUTHORIZED;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
 
 public class SaslTest {
+
   private static final Logger log = LoggerFactory.getLogger(SaslTest.class);
+  private static final String NEHA_BASIC_AUTH = "bmVoYTpha2Zhaw==";
+  private static final String JUN_BASIC_AUTH = "anVuOmthZmthLQ==";
+  private static final String HTTP_URI = "http://localhost:8080";
+  private static final String WS_URI = "ws://localhost:8080/ws";
+  private static final Pattern WS_ERROR_PATTERN = Pattern.compile(".*code=(\\d+).*");
 
-  private File jaasFile;
-  private File loginPropertiesFile;
   private String previousAuthConfig;
   private SaslTestApplication app;
   private CloseableHttpClient httpclient;
-  String httpUri = "http://localhost:8080";
 
   @Before
   public void setUp() throws Exception {
-    jaasFile = File.createTempFile("jaas", ".config");
-    loginPropertiesFile = File.createTempFile("login", ".properties");
+    File jaasFile = File.createTempFile("jaas", ".config");
```

Comment:
we should probably move the tempfiles to use a [junit tempfolder rule](https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html) to avoid littering tmp with files.

Reply:
Sure, I'll take a look.